_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -54,6 +56,35 @@ popular Bitcoin infrastructure software.
     problems that are the result of this bug---but developers are still
     investigating.
 
+- **US Patent 7,110,538 has expired:** Bitcoin transactions are secured using
+  [ECDSA][] (the _Elliptic Curve Digital Signature Algorithm_). Verifying
+  signatures involves multiplying points on the elliptic curve by scalars.
+  Typically, each transaction input requires one or more signature verifications,
+  meaning that syncing the Bitcoin block chain requires many millions of these
+  elliptic curve point multiplications. Any technique to make point
+  multiplications more efficient therefore has the potential to significantly
+  speed up Bitcoin's initial sync.
+
+    In 2011, [Hal Finney described][finney endomorphism] a way to more
+    efficiently compute eliptic curve point multiplications using an
+    [endomorphism][] on the curve (a mapping from the curve to itself which
+    preserves all relationships between points). By using this endomorphism,
+    the multiplication can effectively be broken into two parts, calculated separately,
+    and then recombined to arrive at the solution. Doing this can reduce the
+    number of expensive computations by up to 33%. Finney wrote a proof-of-concept
+    implementation of the endomorphism, which he claimed sped up signature
+    verification by around 25%.
+
+    Pieter Wuille separately implemented the efficient endomorphism algorithm
+    in the [libsecp][] library, which is used to verify signatures in Bitcoin Core.
+    However, the algorithm was encumbered by [U.S. Patent 7,110,538][endomorphism
+    patent] and so the implementation has not previously been compiled or
```

Comment:
Maybe drop "compiled or".  It's a little bit confusing as I'm sure Pieter, if no one else, has compiled it.  :-)  What matters is that it hasn't been distributed, so I think the text can say just that.

Reply:
done